static_src/components/user_list.jsx

```diff
@@ -22,19 +27,59 @@ export default class UserList extends React.Component {
     this.setState({users: nextProps.initialUsers});
   }
 
+  _handleDelete = (userGuid, ev) => {
+    this.props.onRemove(userGuid, ev);
+  }
+
   get columns() {
-    return [
+    var columns = [
       { label: 'Name', key: 'username' },
       { label: 'Date Created', key: 'created_at' }
     ];
+
+    if (this.props.onRemove) {
```

Comment:
Ha so currently it's not set for the space user list and it will add a remove button to the list. I added a note to the UserList propTypes on link 94. I think that's a good place to comment as it's kinda defining the type for the component. Let me know what you think.